src/test/java/com/g4s8/ghman/data/DatabaseExtension.java

```diff
@@ -41,23 +43,41 @@
     private final PGSimpleDataSource src = new PGSimpleDataSource();
 
     /**
-     * Embedded postgres.
+     * Testcontainers postgres.
      */
-    private final EmbeddedPostgres postgres = new EmbeddedPostgres(Version.V11_1);
+    private final PostgreSQLContainer postgres = new PostgreSQLContainer<>()
+        .withUsername("test_user")
+        .withPassword("test_pass");
 
     @Override
     public void beforeEach(final ExtensionContext context) throws Exception {
-        this.src.setUrl(this.postgres.start());
-        Flyway.configure().dataSource(this.src).load().migrate();
+        this.postgres.start();
+        this.src.setUrl(
+            new Joined(
+                "",
+                this.postgres.getJdbcUrl(),
+                "?user=",
+                this.postgres.getUsername(),
+                "&password=",
+                this.postgres.getPassword()
+            ).asString()
+        );
+        new JdbcSession(this.src).sql(
```

Comment:
@scristalli what is wrong with `Flyway` migrations? Will we have to add here each new sql script manually?

Reply:
@olenagerasimova you are absolutely right, major mistake of mine. Restored `Flyway` in commit 2e81d89ea35269f8cdc1f01c8e7b804811c762a8.